### ISSUE TITLE
ADE Display Job Information

### ISSUE THREAD

**Issue by echyam** [MAAP-Project/ZenHub#65]
Users should be able to view job details (ID, algorithm, inputs, etc) from the ADE.  This can be either through the traditional Menu Option -> Dialog or via the UI side panel and main area widget.

Depends on Issue `Update DPS Dockers to report docker stats #62`

**Comment by oddes** [comment]
As per DEV, this is not ready for testing. API needs the MAAP updates to be automatically deployed on terraform.

**Comment by oddes** [comment]
Adding the "DPS UI Menu" to show the ADE Display Job Information is real cool Idea. Following enhancements and minor issues needs to be fixed.
1. Algorithm List is currently showing as plain text, user have no idea that it is clickable, so it should be either a hyperlink or button type functionality
2. When there are no Jobs submitted, currently left panel is showing empty, should display information text that there are no jobs to display.
3. When I submit a job from UI, submitted job information is not showing on the left panel dynamically. Currently, its static and user needs to refresh every time, it's confusing.
4. As shown in the attached screenshot, some input fields are showing twice.
![Screen Shot 2020-08-11 at 4.24.39 PM.png](https://images.zenhubusercontent.com/5efbaf0bd4d83e06b6d4316e/b8b07620-ab19-4002-8bdd-4d492ec10a3a)

**Comment by echyam** [comment]
@oddes 
changes in new `ui-improvements branch` address (1) and (2):
![image](https://user-images.githubusercontent.com/11476825/90179412-44430d80-dd62-11ea-971d-f920f8ac253c.png)

![image](https://user-images.githubusercontent.com/11476825/90180123-5e312000-dd63-11ea-99ed-c0dd138c04de.png)

**Comment by gchang** [comment]
@echyam will make a new issue for auto-refreshing the job status. Currently can perform the refresh by manually clicking refresh.

**Comment by oddes** [comment]
@echyam 
Working in the ui-improvements branch, noticed couple of issues.
1. Following two tabs are not working.
   MAAP Login 
   DPS UI Menu
2. Build failure errors coming:

[W 17:59:20.528 LabApp] 
    "edsc_extension@0.1.0" is not compatible with the current JupyterLab
    Conflicting Dependencies:
    JupyterLab              Extension      Package
    >=1.2.1 <1.3.0          >=2.1.2 <3.0.0 @jupyterlab/application
    >=1.2.1 <1.3.0          >=2.1.1 <3.0.0 @jupyterlab/apputils
    >=1.2.1 <1.3.0          >=2.1.2 <3.0.0 @jupyterlab/docmanager
    >=1.2.2 <1.3.0          >=2.1.2 <3.0.0 @jupyterlab/notebook
    
    
[W 17:59:44.744 LabApp] Build recommended
[W 17:59:44.744 LabApp] dps_info needs to be removed from build
[W 17:59:44.744 LabApp] edsc_extension needs to be removed from build
[W 17:59:44.744 LabApp] insert_defaults_to_notebook needs to be removed from build
[W 17:59:44.744 LabApp] user_meta_form needs to be removed from build
[W 17:59:44.744 LabApp] maapsec needs to be removed from build

Please advise me if there are any changes needs to be made on my end.

**Comment by oddes** [comment]
@echyam @gchang 
Issues 1 and 2 are fixed as expected but I see some issues as mentioned below. It would be great if u can address these in this ticket or an enhancement ticket.

1. algorithms are updated to an hyperlink but when u hover should trigger an event when a user places a mouse over a hyperlink else user does not know its a clickable function, needs to be fixed.
2. As shown in attached screenshot, some times when I double click on any algorithm, it is highlighting all of them, needs to be fixed.

**Comment by oddes** [comment]
Discussed with @echyam to address issues 1 and 2, we will create an enhancement ticket.
